File: src/main/java/ro/hibyte/horus/utils/FilterUtils.java
package ro.hibyte.horus.utils;

import one.space.spo.app.service.contentitem.EntityItemPropertyFilter;

public class FilterUtils {

    public static EntityItemPropertyFilter filterForEquals(String typeKey, String value) {
        return EntityItemPropertyFilter.builder()
                .property(typeKey)
                .filterType(EntityItemPropertyFilter.Type.EQUALS)
                .value(value)
                .build();
    }

}

File: src/main/java/ro/hibyte/horus/utils/FilterUtils.kt
package ro.hibyte.horus.utils

import one.space.spo.app.service.contentitem.EntityItemPropertyFilter

fun filterForEquals(typeKey: String?, value: String?): EntityItemPropertyFilter =
        EntityItemPropertyFilter.builder()
                                .property(typeKey)
                                .filterType(EntityItemPropertyFilter.Type.EQUALS)
                                .value(value)
                                .build()
